code/game/objects/items/weapons/defib.dm

```diff
@@ -489,7 +489,9 @@
 
 						if (H.suiciding || (H.disabilities & NOCLONE))
 							failed = "<span class='warning'>[req_defib ? "[defib]" : "[src]"] buzzes: Resuscitation failed - Recovery of patient impossible. Further attempts futile.</span>"
-						else if ((tplus > tlimit) || !H.getorgan(/obj/item/organ/internal/heart))
+						else if (H.getorgan(/obj/item/organ/internal/heart))
```

Comment:
On defibs, yes. I only seperated the time check from heart missing for more info.